Mage.Sets/src/mage/cards/p/PhantasmalBear.java

```diff
@@ -25,7 +25,9 @@ public PhantasmalBear(UUID ownerId, CardSetInfo setInfo) {
         this.toughness = new MageInt(2);
 
         // When Phantasmal Bear becomes the target of a spell or ability, sacrifice it.
-        this.addAbility(new SourceBecomesTargetTriggeredAbility(new SacrificeSourceEffect()));
+        this.addAbility(new SourceBecomesTargetTriggeredAbility(
+                new SacrificeSourceEffect().setSourceName("it")
```

Comment:
I was trying to handle these automatically in `TriggeredAbilityImpl` but these ones are getting missed right now as the trigger phrase contains "of a". I have some ideas for how to improve the automatic detection. Goal is to avoid having this sort of manual override on the effect.